cmd/porter/explain.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/deislabs/porter/pkg/porter"
+)
+
+func buildBundleExplainCommand(p *porter.Porter) *cobra.Command {
+
+	opts := porter.ExplainOpts{}
+	cmd := cobra.Command{
+		Use:   "explain",
+		Short: "Explain a bundle",
+		Long:  "Explain how to use a bundle by printing the parameters, credentials, outputs, actions and images.",
+		Example: `  porter bundle explain
+  porter bundle explain --file another/porter.yaml
+  porter bundle explain --cnab-file some/bundle.json
+  porter bundle explain --tag deislabs/porter-bundle:v0.1.0
+  porter bundle explain --cnab-file some/bundle.json -o json
```

Comment:
nit: I don't think you need the last two examples as we don't provide examples for the --output flags. They are consistent across all the commands.